From 1e2e7b4c379f1df593269febb62ccbcdd7c892cb Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Fri, 24 Jun 2022 14:03:01 +0000
Subject: [PATCH] panfrost: enable Mali T628 support

Add the T628 chip ID so panfrost runs. Support is experimental!

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 src/panfrost/lib/pan_props.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/panfrost/lib/pan_props.c b/src/panfrost/lib/pan_props.c
index 693b475beb5..cdba8db319e 100644
--- a/src/panfrost/lib/pan_props.c
+++ b/src/panfrost/lib/pan_props.c
@@ -53,6 +53,7 @@
 
 /* Table of supported Mali GPUs */
 const struct panfrost_model panfrost_model_list[] = {
+        MODEL(0x620, "T620", "T62x", NO_ANISO, {}),
         MODEL(0x720, "T720", "T72x", NO_ANISO, { .no_hierarchical_tiling = true }),
         MODEL(0x750, "T760", "T76x", NO_ANISO, {}),
         MODEL(0x820, "T820", "T82x", NO_ANISO, { .no_hierarchical_tiling = true }),
-- 
2.17.1

